readthedocsext/theme/templates/projects/pull_requests_form.html

```diff
@@ -0,0 +1,31 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+{% load static %}
+{% load crispy_forms_filters %}
+
+{% block title %}{% trans "Pull request builds" %}{% endblock %}
+
+{% block project_pull_requests_active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Pull request builds" %}{% endblock %}
+
+{% block project_edit_content %}
+  <p>
+    {% blocktrans trimmed %}
+    Enable builds for your pull requests and easily preview changes to your documentation.
+    {% endblocktrans %}
+  </p>
+
+  <form class="ui form" method="post" action=".">
+    {% csrf_token %}
+    {{ form | crispy }}
+
+    <p>
+      <input class="ui button primary" type="submit" value="{% trans "Update" %}">
+    </p>
```

Comment:
The `p` element is extraneous and can be dropped

```suggestion
    <input class="ui button primary" type="submit" value="{% trans "Update" %}" />
```